test/runTest/server.js

```diff
@@ -399,6 +402,7 @@ async function start() {
                     actualVersion: data.actualVersion,
                     expectedVersion: data.expectedVersion,
                     renderer: data.renderer || '',
+                    useCoarsePointer: data.useCoarsePointer || 'auto',
```

Comment:
If `useCoarsePointer` is `false`?

Reply:
This should be fine because `data.useCoarsePointer` is typed in string. Please check my latest commit to see if it's OK.